go-1.21.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: go-1.21
-  version: 1.21_rc3
-  epoch: 2
+  version: 1.21_rc4
+  epoch: 3
```

Comment:
Do we need the epoch bump here if we're bumping the version?

Reply:
it makes me think that the `epoch`  should be `0` here